Runtime/Game/Requests/PurchaseRequest.cs

```diff
@@ -37,6 +37,41 @@ public class LootLockerAndroidPurchaseVerificationRequest
         public int asset_id { get; set; }
         public string purchase_token { get; set; }
     }
+
+    public class LootLockerPurchaseCatalogItemResponse : LootLockerResponse
+    {
+
```

Comment:
Is this empty on purpose? And if so, that function could use a LootLockerResponse instead?